src/ffi/models/index.js

```diff
@@ -0,0 +1,69 @@
+import ref from 'ref';
+import StructType from 'ref-struct';
+import Enum from 'enum';
+import {
+  refString,
+  usize,
+  u8Arr,
+  u8Pointer,
+  bool
+} from './ref_types';
+
+export const AppKeys = StructType({
+  owner_key: u8Arr,
+  enc_key: u8Arr,
+  sign_pk: u8Arr,
+  sign_sk: u8Arr,
+  enc_pk: u8Arr,
+  enc_sk: u8Arr
+});
+
+export const AppExchangeInfo = StructType({
+  id: refString,
```

Comment:
according to the FFI in safe-libs this is an FFI-String, while this is a `ref.types.string` (so a `CString`). Is that on purpose and because FFI-String is currently broken or is any other reason why this is the wrong type? And does this work right now?